data/json/itemgroups/food_service.json

```diff
@@ -1008,6 +1008,20 @@
       { "item": "sweetbread", "prob": 2, "charges": 2 }
     ]
   },
+  {
+    "type": "item_group",
+    "id": "raw_meat",
```

Comment:
This new item group doesn't seem to be used anywhere.

Reply:
Removed and slated for another PR